blog/01-21-2023-how-to-build-a-basic-crud-app-in-solidity/how-to-build-a-basic-crud-app-in-solidity.md

```diff
@@ -0,0 +1,382 @@
+---
+title: How to build a Basic CRUD App in Solidity
+description: This post will teach you how to create a basic Create, Read, Update and Delete (CRUD) smart contract and test it with the Truffle framework.
+authors:
+  - name: Oyeniyi Abiola Peace
+    title: Community Moderator, Celo
+    url: https://github.com/iamoracle
+    image_url: https://github.com/iamoracle.png
+tags: [CRUD, Celo Sage, Solidity]
```

Comment:
For tags, please refer to the file `src/data-tutorials/users.tsx` in the `TagType` section